<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Publications | Lingfei Wang</title> <meta name="author" content="Lingfei Wang"> <meta name="description" content="Website under construction. "> <link rel="stylesheet" href="/test/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/test/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/test/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/test/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lingfeiwang.github.io/test/publications/"> <link rel="stylesheet" href="/test/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/test/assets/js/theme.js?6185d15ea1982787ad7f435576553d64"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/test/"><span class="font-weight-bold">Lingfei </span>Wang</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/test/">About</a> </li> <li class="nav-item "> <a class="nav-link" href="/test/projects/">Research</a> </li> <li class="nav-item active"> <a class="nav-link" href="/test/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/test/cv/">CV</a> </li> <li class="nav-item "> <a class="nav-link" href="/test/repositories/">Repositories</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_dictys_2023" class="col-sm-8"> <div class="title">Dictys: dynamic gene regulatory network dissects developmental continuum with single-cell multiomics</div> <div class="author"> <em>Lingfei Wang</em>, Nikolaos Trasanidis, Ting Wu, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Guanlan Dong, Michael Hu, Daniel E. Bauer, Luca Pinello' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">4 more authors</span> </div> <div class="periodical"> <em>Nature Methods</em>, Aug 2023 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Gene regulatory networks (GRNs) are key determinants of cell function and identity and are dynamically rewired during development and disease. Despite decades of advancement, challenges remain in GRN inference, including dynamic rewiring, causal inference, feedback loop modeling and context specificity. To address these challenges, we develop Dictys, a dynamic GRN inference and analysis method that leverages multiomic single-cell assays of chromatin accessibility and gene expression, context-specific transcription factor footprinting, stochastic process network and efficient probabilistic modeling of single-cell RNA-sequencing read counts. Dictys improves GRN reconstruction accuracy and reproducibility and enables the inference and comparative analysis of context-specific and dynamic GRNs across developmental contexts. Dictys’ network analyses recover unique insights in human blood and mouse skin development with cell-type-specific and dynamic GRNs. Its dynamic network visualizations enable time-resolved discovery and investigation of developmental driver transcription factors and their regulated targets. Dictys is available as a free, open-source and user-friendly Python package.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="bankier_plasma_2023" class="col-sm-8"> <div class="title">Plasma cortisol-linked gene networks in hepatic and adipose tissues implicate corticosteroid-binding globulin in modulating tissue glucocorticoid action and cardiovascular risk</div> <div class="author"> Sean Bankier, <em>Lingfei Wang</em>, Andrew Crawford, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Ruth A. Morgan, Arno Ruusalepp, Ruth Andrew, Johan L. M. Björkegren, Brian R. Walker, Tom Michoel' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">6 more authors</span> </div> <div class="periodical"> <em>Frontiers in Endocrinology</em>, Aug 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Genome-wide association meta-analysis (GWAMA) by the Cortisol Network (CORNET) consortium identified genetic variants spanning the SERPINA6/SERPINA1 locus on chromosome 14 associated with morning plasma cortisol, cardiovascular disease (CVD), and SERPINA6 mRNA expression encoding corticosteroid-binding globulin (CBG) in the liver. These and other findings indicate that higher plasma cortisol levels are causally associated with CVD; however, the mechanisms by which variations in CBG lead to CVD are undetermined. Using genomic and transcriptomic data from The Stockholm Tartu Atherosclerosis Reverse Networks Engineering Task (STARNET) study, we identified plasma cortisol-linked single-nucleotide polymorphisms (SNPs) that are trans-associated with genes from seven different vascular and metabolic tissues, finding the highest representation of trans-genes in the liver, subcutaneous fat, and visceral abdominal fat, [false discovery rate (FDR) = 15%]. We identified a subset of cortisol-associated trans-genes that are putatively regulated by the glucocorticoid receptor (GR), the primary transcription factor activated by cortisol. Using causal inference, we identified GR-regulated trans-genes that are responsible for the regulation of tissue-specific gene networks. Cis-expression Quantitative Trait Loci (eQTLs) were used as genetic instruments for identification of pairwise causal relationships from which gene networks could be reconstructed. Gene networks were identified in the liver, subcutaneous fat, and visceral abdominal fat, including a high confidence gene network specific to subcutaneous adipose (FDR = 10%) under the regulation of the interferon regulatory transcription factor, IRF2. These data identify a plausible pathway through which variation in the liver CBG production perturbs cortisol-regulated gene networks in peripheral tissues and thereby promote CVD.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="you_qrich1_2021" class="col-sm-8"> <div class="title">QRICH1 dictates the outcome of ER stress through transcriptional control of proteostasis</div> <div class="author"> Kwontae You, <em>Lingfei Wang</em>, Chih-Hung Chou, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Kai Liu, Toru Nakata, Alok Jaiswal, Junmei Yao, Ariel Lefkovith, Abdifatah Omar, Jacqueline G. Perrigoue, Jennifer E. Towne, Aviv Regev, Daniel B. Graham, Ramnik J. Xavier' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">11 more authors</span> </div> <div class="periodical"> <em>Science</em>, Jan 2021 </div> <div class="periodical"> Publisher: American Association for the Advancement of Science Section: Research Article </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Transcriptional control of proteostasis Tissue homeostasis requires the coordinated activity of multiple cell types to initiate and then resolve inflammation. Intrinsic cellular stress-response pathways facilitate adaptation to stress and tissue restitution. Among these stress pathways, the unfolded protein response can elicit two divergent outcomes: adaptation to endoplasmic reticulum (ER) stress or termination by programmed cell death. You et al. identified QRICH1 as a transcriptional regulator controlling adaptation to ER stress at the level of protein translation and secretion. The authors further demonstrate the role of the QRICH1 program in inflammatory diseases of the colon and liver. Science, this issue p. 45 Structured Abstract INTRODUCTIONTissue homeostasis requires the coordinated activity of multiple cell types to initiate and then resolve inflammation. Endoplasmic reticulum (ER) stress is a hallmark of inflammation and exacerbates tissue pathology across a broad range of human diseases. Environmental stressors associated with inflammation and cell-intrinsic metabolic demands can elicit ER stress, protein misfolding, and cell death. To counteract these processes, stress response pathways, including the unfolded protein response (UPR), facilitate adaptation to stress and tissue restitution. Cells sense ER stress and initiate the UPR through three coordinated pathways mediated by the effector proteins inositol-requiring enzyme 1 (IRE1/ERN1), activating transcription factor 6 (ATF6), and protein kinase RNA-like ER kinase (PERK,EIF2AK3). Collectively, the UPR effector pathways fine-tune the rate of protein translation and induce transcriptional up-regulation of genes that promote ER function, such as those encoding chaperone proteins and secretory machinery. Although these functional responses to ER stress by the UPR pathways aim to restore cellular homeostasis, prolonged and unresolved ER stress can elicit programed cell death. In this context, the molecular mechanisms that dictate the outcome of ER stress are incompletely understood. RATIONALEMismanagement of ER stress in intestinal epithelial cells can lead to disruption of barrier integrity, resulting in exposure of the host immune sysem to commensal microbes that trigger uncontrolled inflammation. With accumulating evidence highlighting the prominent role of ER stress in disease, it remains to be determined how the UPR directs divergent cell fate decisions. The UPR either induces an adaptive phase that promotes recovery of ER proteostasis and cell survival or induces a terminal phase that initiates the active engagement of programmed cell death pathways. RESULTSToward the objective of defining mechanisms controlling the adaptive versus terminal UPR, we used single-cell RNA sequencing (scRNA-seq) in primary intestinal epithelial monolayers. Single-cell resolution enabled detailed kinetic profiling of dynamic transcriptional states that correspond to the early acute UPR followed by adaptive restoration of ER homeostasis or terminal cell death. In parallel, we performed a genome-wide CRISPR screen to identify regulatory nodes that control the terminal UPR. Integrative analysis of CRISPR screen results with single-cell transcriptional profiling identified QRICH1 as a critical determinant of cellular entry into the terminal versus adaptive UPR. We demonstrate that QRICH1 is a key effector of the PERK-eIF2α axis of the UPR and that its translation is regulated by an upstream open reading frame in the QRICH1 mRNA. Using a combination of RNA-seq and chromatin immunoprecipitation sequencing (ChIP-seq), we show that QRICH1 bound promoter regions to control a transcriptional module that regulates protein translation and secretory networks. QRICH1-mediated translational activation increased protein flux into the ER and proteotoxicity, whereas QRICH1 knockout protected intestinal epithelial cells from proteotoxicity. Finally, to assess the role of QRICH1 in human disease, we analyzed biopsies of patients with ulcerative colitis (UC) and found evidence of enrichment of the QRICH1 transcriptional signature in inflamed colon biopsies, particularly in secretory epithelial cells and enterocytes. The QRICH1 transcriptional signature was also up-regulated in biopsies of patients with nonalcoholic steatohepatitis and in inflamed and cirrhotic samples from liver biopsies. CONCLUSIONHere, we identify a distinct arm of the PERK-eIF2α axis mediated by the transcriptional regulator QRICH1. Cells dynamically respond to ER stress by inducing up-regulation of QRICH1, which modulates translation and transit of proteins through the ER-Golgi secretory pathway. Thus, QRICH1 acts as a regulator of a distinct transcriptional module that coordinates cellular stress responses to regulate protein synthesis and secretion under homeostatic and pathological conditions. Taken together, these findings suggest a broadly conserved role for the QRICH1 transcriptional program in managing cell stress responses and acting as a gatekeeper for controlling cellular entry into the adaptive versus terminal UPR. Mechanistic characterization of QRICH1 within this context provides insight into how cells manage responses to stress and expands our understanding of the UPR pathway, broadening our understanding of the molecular mechanisms by which cellular stress responses are dynamically regulated. \textlessimg class="fragment-image" aria-describedby="F1-caption" src="https://science-sciencemag-org.ezp-prod1.hul.harvard.edu/content/sci/371/6524/eabb6896/F1.medium.gif"/\textgreater Download high-res image Open in new tab Download Powerpoint QRICH1 controls a distinct arm of the PERK-eIF2α axis to modulate proteostasis and dictate entry into the adaptive versus terminal UPR.In response to ER stress, PERK phosphorylates eIF2α, suppressing global translation while promoting ATF4 and QRICH1 translation by bypassing inhibitory upstream open reading frames (uORFs). QRICH1 localizes to the nucleus and positively regulates the transcription of genes that regulate protein secretion. Prolonged QRICH1 expression is associated with proteotoxicity and cell death during the terminal UPR, whereas its down-regulation is associated with restoration of ER homeostasis during the adaptive UPR. P, phosphorylation; p-PERK, phosphorylated PERK; p-eIF2α, phosphorylated eIF2α. Tissue homeostasis is perturbed in a diversity of inflammatory pathologies. These changes can elicit endoplasmic reticulum (ER) stress, protein misfolding, and cell death. ER stress triggers the unfolded protein response (UPR), which can promote recovery of ER proteostasis and cell survival or trigger programmed cell death. Here, we leveraged single-cell RNA sequencing to define dynamic transcriptional states associated with the adaptive versus terminal UPR in the mouse intestinal epithelium. We integrated these transcriptional programs with genome-scale CRISPR screening to dissect the UPR pathway functionally. We identified QRICH1 as a key effector of the PERK-eIF2α axis of the UPR. QRICH1 controlled a transcriptional program associated with translation and secretory networks that were specifically up-regulated in inflammatory pathologies. Thus, QRICH1 dictates cell fate in response to pathological ER stress. A glutamine-rich protein regulates translation and secretory networks that are specifically up-regulated in inflammatory pathologies. A glutamine-rich protein regulates translation and secretory networks that are specifically up-regulated in inflammatory pathologies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_current_2021" class="col-sm-8"> <div class="title">Current progress and potential opportunities to infer single-cell developmental trajectory and cell fate</div> <div class="author"> <em>Lingfei Wang</em>, Qian Zhang, Qian Qin, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Nikolaos Trasanidis, Michael Vinyard, Huidong Chen, Luca Pinello' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">4 more authors</span> </div> <div class="periodical"> <em>Current Opinion in Systems Biology</em>, Mar 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Rapid technological advances in transcriptomics and lineage tracing technologies provide new opportunities to understand organismal development at the single-cell level. Building on these advances, various computational methods have been proposed to infer developmental trajectories and to predict cell fate. These methods have unveiled previously uncharacterized transitional cell types and differentiation processes. Importantly, the ability to recover cell states and trajectories has been evolving hand-in-hand with new technologies and diverse experimental designs; more recent methods can capture complex trajectory topologies and infer short- and long-term cell fate dynamics. Here, we summarize and categorize the most recent and popular computational approaches for trajectory inference based on the information they leverage and describe future challenges and opportunities for the development of new methods for reconstructing differentiation trajectories and inferring cell fates.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_single-cell_2021" class="col-sm-8"> <div class="title">Single-cell normalization and association testing unifying CRISPR screen and gene co-expression analyses with Normalisr</div> <div class="author"> <em>Lingfei Wang</em> </div> <div class="periodical"> <em>Nature Communications</em>, Nov 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Single-cell RNA sequencing (scRNA-seq) provides unprecedented technical and statistical potential to study gene regulation but is subject to technical variations and sparsity. Furthermore, statistical association testing remains difficult for scRNA-seq. Here we present Normalisr, a normalization and statistical association testing framework that unifies single-cell differential expression, co-expression, and CRISPR screen analyses with linear models. By systematically detecting and removing nonlinear confounders arising from library size at mean and variance levels, Normalisr achieves high sensitivity, specificity, speed, and generalizability across multiple scRNA-seq protocols and experimental conditions with unbiased p-value estimation. The superior scalability allows us to reconstruct robust gene regulatory networks from trans-effects of guide RNAs in large-scale single cell CRISPRi screens. On conventional scRNA-seq, Normalisr recovers gene-level co-expression networks that recapitulated known gene functions.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="carey_tfeb_2020" class="col-sm-8"> <div class="title">TFEB Transcriptional Responses Reveal Negative Feedback by BHLHE40 and BHLHE41</div> <div class="author"> Kimberly L. Carey, Geraldine L. C. Paulus, <em>Lingfei Wang</em>, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Dale R. Balce, Jessica W. Luo, Phil Bergman, Ianina C. Ferder, Lingjia Kong, Nicole Renaud, Shantanu Singh, Maria Kost-Alimova, Beat Nyfeler, Kara G. Lassen, Herbert W. Virgin, Ramnik J. Xavier' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">12 more authors</span> </div> <div class="periodical"> <em>Cell Reports</em>, Nov 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Transcription factor EB (TFEB) activates lysosomal biogenesis genes in response to environmental cues. Given implications of impaired TFEB signaling and lysosomal dysfunction in metabolic, neurological, and infectious diseases, we aim to systematically identify TFEB-directed circuits by examining transcriptional responses to TFEB subcellular localization and stimulation. We reveal that steady-state nuclear TFEB is sufficient to activate transcription of lysosomal, autophagy, and innate immunity genes, whereas other targets require higher thresholds of stimulation. Furthermore, we identify shared and distinct transcriptional signatures between mTOR inhibition and bacterial autophagy. Using a genome-wide CRISPR library, we find TFEB targets that protect cells from or sensitize cells to lysosomal cell death. BHLHE40 and BHLHE41, genes responsive to high, sustained levels of nuclear TFEB, act in opposition to TFEB upon lysosomal cell death induction. Further investigation identifies genes counter-regulated by TFEB and BHLHE40/41, adding this negative feedback to the current understanding of TFEB regulatory mechanisms.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_accurate_2019" class="col-sm-8"> <div class="title">Accurate wisdom of the crowd from unsupervised dimension reduction</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>Royal Society Open Science</em>, Nov 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Wisdom of the crowd, the collective intelligence from responses of multiple human or machine individuals to the same questions, can be more accurate than each individual and improve social decision-making and prediction accuracy. Crowd wisdom estimates each individual’s error level and minimizes the overall error in the crowd consensus. However, with problem-specific models mostly concerning binary (yes/no) predictions, crowd wisdom remains overlooked in biomedical disciplines. Here we show, in real-world examples of transcription factor target prediction and skin cancer diagnosis, and with simulated data, that the crowd wisdom problem is analogous to one-dimensional unsupervised dimension reduction in machine learning. This provides a natural class of generalized, accurate and mature crowd wisdom solutions, such as PCA and Isomap, that can handle binary and also continuous responses, like confidence levels. They even outperform supervised-learning-based collective intelligence that is calibrated on historical performance of individuals, e.g. random forest. This study unifies crowd wisdom and unsupervised dimension reduction, and extends its applications to continuous data. As the scales of data acquisition and processing rapidly increase, especially in high-throughput sequencing and imaging, crowd wisdom can provide accurate predictions by combining multiple datasets and/or analytical methods.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_high-dimensional_2019" class="col-sm-8"> <div class="title">High-Dimensional Bayesian Network Inference From Systems Genetics Data Using Genetic Node Ordering</div> <div class="author"> <em>Lingfei Wang</em>, Pieter Audenaert, and Tom Michoel</div> <div class="periodical"> <em>Frontiers in Genetics</em>, Nov 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Studying the impact of genetic variation on gene regulatory networks is essential to understand the biological mechanisms by which genetic variation causes variation in phenotypes. Bayesian networks provide an elegant statistical approach for multi-trait genetic mapping and modelling causal trait relationships. However, inferring Bayesian gene networks from high-dimensional genetics and genomics data is challenging, because the number of possible networks scales super-exponentially with the number of nodes, and the computational cost of conventional Bayesian network inference methods quickly becomes prohibitive. We propose an alternative method to infer high-quality Bayesian gene networks that easily scales to thousands of genes. Our method first reconstructs a node ordering by conducting pairwise causal inference tests between genes, which then allows to infer a Bayesian network via a series of independent variable selection problems, one for each gene. We demonstrate using simulated and real systems genetics data that this results in a Bayesian network with equal, and sometimes better, likelihood than the conventional methods, while having a significantly higher overlap with groundtruth networks and being orders of magnitude faster. Moreover our method allows for a unified false discovery rate control across genes and individual edges, and thus a rigorous and easily interpretable way for tuning the sparsity level of the inferred network. Bayesian network inference using pairwise node ordering is a highly efficient approach for reconstructing gene regulatory networks when prior information for the inclusion of edges exists or can be inferred from the available data.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="vipin_causal_2018" class="col-sm-8"> <div class="title">Causal Transcription Regulatory Network Inference Using Enhancer Activity as a Causal Anchor</div> <div class="author"> Deepti Vipin, <em>Lingfei Wang</em>, Guillaume Devailly, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Tom Michoel, Anagha Joshi' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '3'); ">2 more authors</span> </div> <div class="periodical"> <em>International Journal of Molecular Sciences</em>, Nov 2018 </div> <div class="periodical"> Number: 11 Publisher: Multidisciplinary Digital Publishing Institute </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Transcription control plays a crucial role in establishing a unique gene expression signature for each of the hundreds of mammalian cell types. Though gene expression data have been widely used to infer cellular regulatory networks, existing methods mainly infer correlations rather than causality. We developed statistical models and likelihood-ratio tests to infer causal gene regulatory networks using enhancer RNA (eRNA) expression information as a causal anchor and applied the framework to eRNA and transcript expression data from the FANTOM Consortium. Predicted causal targets of transcription factors (TFs) in mouse embryonic stem cells, macrophages and erythroblastic leukaemia overlapped significantly with experimentally-validated targets from ChIP-seq and perturbation data. We further improved the model by taking into account that some TFs might act in a quantitative, dosage-dependent manner, whereas others might act predominantly in a binary on/off fashion. We predicted TF targets from concerted variation of eRNA and TF and target promoter expression levels within a single cell type, as well as across multiple cell types. Importantly, TFs with high-confidence predictions were largely different between these two analyses, demonstrating that variability within a cell type is highly relevant for target prediction of cell type-specific factors. Finally, we generated a compendium of high-confidence TF targets across diverse human cell and tissue types.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_comparable_2017" class="col-sm-8"> <div class="title">Comparable variable selection with Lasso</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>arXiv:1701.07011 [q-bio, stat]</em>, Jan 2017 </div> <div class="periodical"> arXiv: 1701.07011 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>P-values are being computed for increasingly complicated statistics but lacking evaluations on their quality. Meanwhile, accurate p-values enable significance comparison across batches of hypothesis tests and consequently unified false discover rate (FDR) control. This article discusses two related questions in this setting. First, we propose statistical tests to evaluate the quality of p-value and the cross-batch comparability of any other statistic. Second, we propose a lasso based variable selection statistic, based on when the predictor variable first becomes active, and compute its p-value to achieve unified FDR control across multiple selections. In the end, we apply our tests on covTest, selectiveInference, and our statistic, based on real and null datasets for network inference in normal and high-dimensional settings. Results demonstrate higher p-value quality from our statistic and reveal p-value errors from others hidden before. We implement our statistic as lassopv in R.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_whole-transcriptome_2017" class="col-sm-8"> <div class="title">Whole-transcriptome causal network inference with genomic and transcriptomic data</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>bioRxiv</em>, Nov 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Reconstruction of causal gene networks can distinguish regulators from targets and reduce false positives by integrating genetic variations. Its recent developments in speed and accuracy have enabled whole-transcriptome causal network inference on a personal computer. Here we demonstrate this technique with program Findr on 3,000 genes from the Geuvadis dataset. Subsequent analysis reveals major hub genes in the reconstructed network.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_efficient_2017" class="col-sm-8"> <div class="title">Efficient and accurate causal inference with hidden confounders from genome-transcriptome variation data</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>PLOS Computational Biology</em>, Aug 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Author summary Understanding how genetic variation between individuals determines variation in observable traits or disease risk is one of the core aims of genetics. It is known that genetic variation often affects gene regulatory DNA elements and directly causes variation in expression of nearby genes. This effect in turn cascades down to other genes via the complex pathways and gene interaction networks that ultimately govern how cells operate in an ever changing environment. In theory, when genetic variation and gene expression levels are measured simultaneously in a large number of individuals, the causal effects of genes on each other can be inferred using statistical models similar to those used in randomized controlled trials. We developed a novel method and ultra-fast software Findr which, unlike existing methods, takes into account the complex but unknown network context when predicting causality between specific gene pairs. Findr’s predictions have a significantly higher overlap with known gene networks compared to existing methods, using both simulated and real data. Findr is also nearly a million times faster, and hence the only software in its class that can handle modern datasets where the expression levels of ten-thousands of genes are simultaneously measured in hundreds to thousands of individuals.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_controlling_2017" class="col-sm-8"> <div class="title">Controlling false discoveries in Bayesian gene networks with lasso regression p-values</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>arXiv:1701.07011 [q-bio, stat]</em>, Jan 2017 </div> <div class="periodical"> arXiv: 1701.07011 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bayesian networks can represent directed gene regulations and therefore are favored over co-expression networks. However, hardly any Bayesian network study concerns the false discovery control (FDC) of network edges, leading to low accuracies due to systematic biases from inconsistent false discovery levels in the same study. We design four empirical tests to examine the FDC of Bayesian networks from three p-value based lasso regression variable selections — two existing and one we originate. Our method, lassopv, computes p-values for the critical regularization strength at which a predictor starts to contribute to lasso regression. Using null and Geuvadis datasets, we find that lassopv obtains optimal FDC in Bayesian gene networks, whilst existing methods have defective p-values. The FDC concept and tests extend to most network inference scenarios and will guide the design and improvement of new and existing methods. Our novel variable selection method with lasso regression also allows FDC on other datasets and questions, even beyond network inference and computational biology. Lassopv is implemented in R and freely available at https://github.com/lingfeiwang/lassopv and https://cran.r-project.org/package=lassopv</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_detection_2016" class="col-sm-8"> <div class="title">Detection of Regulator Genes and eQTLs in Gene Networks</div> <div class="author"> <em>Lingfei Wang</em>, and Tom Michoel</div> <div class="periodical"> <em>In Systems Biology in Animal Production and Health, Vol. 1</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Genetic differences between individuals associated to quantitative phenotypic traits, including disease states, are usually found in noncoding genomic regions. These genetic variants are often also associated to differences in expression levels of nearby genes (they are “expression quantitative trait loci” or eQTLs, for short) and presumably play a gene regulatory role, affecting the status of molecular networks of interacting genes, proteins, and metabolites. Computational systems biology approaches to reconstruct causal gene networks from large-scale omics data have therefore become essential to understand the structure of networks controlled by eQTLs together with other regulatory genes, as well as to generate detailed hypotheses about the molecular mechanisms that lead from genotype to phenotype. Here we review the main analytical methods and software to identify eQTLs and their associated genes, to reconstruct coexpression networks and modules, to reconstruct causal Bayesian gene and module networks, and to validate predicted networks in silico.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="li_bound_2014" class="col-sm-8"> <div class="title">Bound to bounce: A coupled scalar–tachyon model for a smooth bouncing/cyclic universe</div> <div class="author"> Changhong Li, <em>Lingfei Wang</em>, and Yeuk-Kwan E. Cheung</div> <div class="periodical"> <em>Physics of the Dark Universe</em>, Apr 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We introduce a string-inspired model for a bouncing/cyclic universe, utilizing the scalar–tachyon coupling as well as contribution from curvature in a closed universe. The universe undergoes the locked inflation, tachyon matter dominated rolling expansion, turnaround and contraction, as well as the subsequent deflation and “bounce” in each cycle of the cosmological evolution. We perform extensive analytic and numerical studies of the above evolution process. The minimum size of the universe is nonzero for generic initial values. The smooth bounces are made possible because of the negative contribution to effective energy density by the curvature term. No ghosts are ever generated at any point in the entire evolution of the universe, with the Null, Weak, and Dominant Energy Conditions preserved even at the bounce points, contrary to many bounce models previously proposed. And the Strong Energy Condition is satisfied in periods with tachyon matter domination.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_cosmological_2013" class="col-sm-8"> <div class="title">Cosmological perturbations from a spectator field during inflation</div> <div class="author"> <em>Lingfei Wang</em>, and Anupam Mazumdar</div> <div class="periodical"> <em>Journal of Cosmology and Astroparticle Physics</em>, Apr 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In this paper we will discuss analytically the perturbations created from a slowly rolling subdominant spectator field which decays much before the end of inflation. The quantum fluctuations of such a spectator field can seed perturbations on very large scales and explain the temperature anisotropy in the cosmic microwave background radiation with moderate non-Gaussianity, provided the relevant modes leave the Hubble patch while the spectator is slowly rolling. Furthermore, the perturbations are purely adiabatic since the inflaton decay dominates and creates all the Standard Model degrees of freedom. We will provide two examples for the spectator field potential, one with a step function profile, and the other with an inflection point. In both the cases we will compute higher order curvature perturbations, i.e. local bispectrum and trispectrum, which can be constrained by the forthcoming Planck data.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="mazumdar_creating_2013" class="col-sm-8"> <div class="title">Creating perturbations from a decaying field during inflation</div> <div class="author"> Anupam Mazumdar, and <em>Lingfei Wang</em> </div> <div class="periodical"> <em>Physical Review D</em>, Apr 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Typically, the fluctuations generated from a decaying field during inflation do not contribute to the large scale structures. In this paper, we provide an example where it is possible for a field which slowly rolls and then decays during inflation to create all the matter perturbations with a slightly red-tilted spectral index, with no isocurvature perturbations, and with a possibility of a departure from Gaussian fluctuations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_small_2013" class="col-sm-8"> <div class="title">Small non-Gaussianity and dipole asymmetry in the cosmic microwave background</div> <div class="author"> <em>Lingfei Wang</em>, and Anupam Mazumdar</div> <div class="periodical"> <em>Physical Review D</em>, Jul 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In this paper we provide a prescription for obtaining a small non-Gaussianity and the observed dipole asymmetry in the cosmic microwave background radiation. The observations inevitably lead to multifield inflationary dynamics, where each field can create positive or negative large non-Gaussianity, resulting in a fine cancellation but with an observable imprint on the hemispherical asymmetry. We discuss this possibility within a simple slow-roll scenario and find that it is hard to explain the observed dipole asymmetry. We briefly discuss some speculative scenarios where one can explain dipole asymmetry.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_visible_2013" class="col-sm-8"> <div class="title">Visible sector inflation and the right thermal history in light of Planck data</div> <div class="author"> <em>Lingfei Wang</em>, Ernestas Pukartas, and Anupam Mazumdar</div> <div class="periodical"> <em>Journal of Cosmology and Astroparticle Physics</em>, Jul 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Inflation creates perturbations for the large scale structures in the universe, but it also dilutes everything. Therefore it is pertinent that the end of inflation must explain how to excite the Standard Model dof along with the dark matter. In this paper we will briefly discuss the role of visible sector inflaton candidates which are embedded within the Minimal Supersymmetric Standard Model (MSSM) and discuss their merit on how well they match the current data from the Planck. Since the inflaton carries the Standard Model charges their decay naturally produces all the relevant dof with no dark/hidden sector radiation and no isocurvature fluctuations. We will first discuss a single supersymmetric flat direction model of inflation and demonstrate what parameter space is allowed by the Planck and the LHC. We will also consider where the perturbations are created by another light field which decays after inflation, known as a curvaton . The late decay of the curvaton can create observable non-Gaussianity. In the end we will discuss the role of a spectator field whose origin may not lie within the visible sector physics, but its sheer presence during inflation can still create all the perturbations responsible for the large scale structures including possible non-Gaussianity, while the inflaton is embedded within the visible sector which creates all the relevant matter including dark matter, but no dark radiation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="mazumdar_cmb_2013" class="col-sm-8"> <div class="title">CMB dipole asymmetry from a fast roll phase</div> <div class="author"> Anupam Mazumdar, and <em>Lingfei Wang</em> </div> <div class="periodical"> <em>Journal of Cosmology and Astroparticle Physics</em>, Jul 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The observed CMB (cosmic microwave background) dipole asymmetry cannot be explained by a single field model of inflation - it inevitably requires more than one field where one of the fields is responsible for amplifying the super-Hubble fluctuations beyond the pivot scale. Furthermore the current constraints on f NL and τ NL require that such an amplification cannot produce large non-Gaussianity. In this paper we propose a model to explain this dipole asymmetry from a spectator field, which is responsible for generating all the curvature perturbations, but has a temporary fast roll phase before the Hubble exit of the pivot scale. The current data prefers spectator scenario because it leaves no isocurvature perturbations. The spectator model will also satisfy the well-known constraints arising from quasars, and the quadrupole and octupole of the CMB.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="mazumdar_separable_2012" class="col-sm-8"> <div class="title">Separable and non-separable multi-field inflation and large non-Gaussianity</div> <div class="author"> Anupam Mazumdar, and <em>Lingfei Wang</em> </div> <div class="periodical"> <em>Journal of Cosmology and Astroparticle Physics</em>, Jul 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In this paper we provide a general framework based on δ N formalism to estimate the cosmological observables pertaining to the cosmic microwave background radiation for non-separable potentials, and for generic end of inflation boundary conditions. We provide analytical and numerical solutions to the relevant observables by decomposing the cosmological perturbations along the curvature and the isocurvature directions, instead of adiabatic and entropy directions . We then study under what conditions large bi-spectrum and tri-spectrum can be generated through phase transition which ends inflation. In an illustrative example, we show that large f NL ##IMG## [http://ej.iop.org/icons/Entities/calO.gif] Script O (80) and τ NL ##IMG## [http://ej.iop.org/icons/Entities/calO.gif] Script O (20000) can be obtained for the case of separable and non-separable inflationary potentials.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="wang_preheating_2011" class="col-sm-8"> <div class="title">Preheating and locked inflation: an analytic approach towards parametric resonance</div> <div class="author"> <em>Lingfei Wang</em> </div> <div class="periodical"> <em>Journal of Cosmology and Astroparticle Physics</em>, Jul 2011 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We take an analytic approach towards the framework of parametric resonance and apply it on preheating and locked inflation. A two-scalar toy model is analytically solved for the λ ##IMG## [http://ej.iop.org/icons/Entities/phi.gif] phi 2 χ 2 coupling for the homogenous modes. The effects of dynamic universe background and backreaction are taken into account. We show the average effect of parametric resonance to be that χ’s amplitude doubles for each cycle of ##IMG## [http://ej.iop.org/icons/Entities/phi.gif] phi . Our framework partly solves the broad resonance for preheating scenario, showing two distinct stages of preheating and making the parameters of preheating analytically calculable. It is demonstrated for slowroll inflation models, preheating is terminated, if by backreaction, typically in the 5th e-fold. Under our framework, a possible inhomogeneity amplification effect is also found during preheating, which both may pose strong constraints on some inflationary models and may amplify tiny existing inhomogeneities to the desired scale. For demonstration, we show it rules out the backreaction end of preheating of the quadratic slowroll inflation model with mass m 10 −6 . For locked inflation, parametric resonance is found to be inhibited if ##IMG## [http://ej.iop.org/icons/Entities/phi.gif] phi has more than one real component.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2024 Lingfei Wang. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/test/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/test/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/test/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/test/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/test/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/test/assets/js/copy_code.js?9b43d6e67ddc7c0855b1478ee4c48c2d" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>